doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,90 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps for denoising of diffusion-weighted MR images.
```

Comment:
This tutorial walks through the steps to denoise diffusion-weighted MR images using DIPY.

Reply:
Reworded in d49dc04